Svc/FileDownlink/FileDownlink.hpp

```diff
@@ -193,7 +193,7 @@ class FileDownlink final : public FileDownlinkComponentBase {
     //! Sources of send file requests
     enum CallerSource { COMMAND, PORT };
 
-#define FILE_ENTRY_FILENAME_LEN 101
```

Comment:
We should switch this one to the FPP constant for file names, rather than having a separate (unequal) filename length